tkdesigner/cli.py

```diff
@@ -0,0 +1,70 @@
+"""
+TKinter Designer command-line interface.
+"""
+
+import os
+import json
+import shutil
+import logging
+import argparse
+from tkdesigner.constants import ASSETS_PATH
+from tkdesigner.parse import FigmaParser
+
+from tkdesigner import figma_api
+
+
+if int(os.getenv("TKDESIGNER_VERBOSE", 0)) == 1:
+    log_level = logging.INFO
+else:
+    log_level = logging.WARN
+
+logging.basicConfig(level=log_level)
+
+
+def create_dirs(output_path, asset_path):
+    os.mkdir(output_path)
+    os.mkdir(asset_path)
+
+
+def main():
+    # Parse command-line arguments
+    parser = argparse.ArgumentParser(description="Generate TKinter GUI code from Figma design.")
+    parser.add_argument("file_url", type=str, help="File url of the Figma design.")
+    parser.add_argument("token", type=str, help="Figma token.")
+    parser.add_argument("-o", "--output", type=str, default="build/", help="Folder to output code and image assets to. Defaults to build/.")
+    parser.add_argument("-f", "--force", action="store_true", help="If this flag is passed in, the output directory given will be overwritten if it exists.")
```

Comment:
This does not seem to work. Even if the directory exists it overrides it without any prompt even if -f is not passed.

Reply:
How is one supposed to execute the cli.py module?
I get `ModuleNotFoundError: No module named 'tkdesigner'` when I execute it from inside the `tkdesigner` directory or from its parent directory `Tkinter-Designer`.